internal/client.go

```diff
@@ -657,32 +657,33 @@ type (
 		// Optional: defaulted to 10 secs.
 		WorkflowTaskTimeout time.Duration
 
-		// WorkflowIDReusePolicy - Specifies server behavior if a *completed* workflow with the same id exists.
-		// This can be useful for dedupe logic if set to RejectDuplicate
+		// WorkflowIDConflictPolicy - Specifies server behavior if a *running* workflow with the same id exists.
+		// This cannot be set if WorkflowIDReusePolicy is set to TerminateIfRunning.
```

Comment:
Conflict policy moved ahead of reuse policy in order because it is logically prior.